packages/app-store/nextcloudtalk/DESCRIPTION.md

```diff
@@ -0,0 +1,7 @@
+---
+items:
+  - nextcloudtalk1.jpg
+  - nextcloudtalk2.jpg
+---
+
+{DESCRIPTION}
```

Comment:
Hey, please add a detailed description of how to retrieve the keys that we are adding, step by step. Take our zoom app description as a reference

Reply:
Can you elaborate on that? There is no description here about retrieving the keys for Zoom: https://github.com/calcom/cal.com/blob/main/packages/app-store/zoomvideo/DESCRIPTION.md

Reply:
Yeah, sorry, it's not in the description, but please create a README file for that Take reference:- https://github.com/calcom/cal.com/blob/main/packages/app-store/webex/README.md

Reply:
Just added a README file :heavy_check_mark:

Reply:
Thanks @CodeShakingSheep. Lastly, can you provide a complete Loom video where you show everything about the workflow?

Reply:
I don't have a Loom account. So, I recorded my screen locally. Here is the link to the video: https://e.pcloud.link/publink/show?code=XZJhYCZE9glp5vsXffQAfvwEmQEK8Urv270
It shows the following:
- Adding an OAuth client in Nextcloud for Cal.com
- Setting up the Nextcloud Talk app in Cal.com (including setup of client ID and secret)
- Activating Nextcloud Talk app for a Cal.com user (including authentication via Nextcloud)
- Setting Nextcloud Talk as location for an event
- Automatic conversation creation in Nextcloud Talk for a booked event
- Automatic conversation removal in Nextcloud Talk for a cancelled event

Reply:
Excellent work, @CodeShakingSheep. Is the location link a redirect for conversion? If so, please prioritize the video or meeting link instead. This means that when a booking is made, it should generate a Nextcloud meeting link

Reply:
The location link is a direct link to the Nextcloud Talk call, like `<NEXTCLOUD_URL>/call/<ID>` See screenshot from the demo booking I created in the video.
![image](https://github.com/user-attachments/assets/d908f2cf-30b0-4bf8-9a11-b68fd6d2ab16)